demo/api/src/importer/importers/product-importer/product-importer.service.ts

```diff
@@ -0,0 +1,16 @@
+import { EntityManager } from "@mikro-orm/core";
+import { Injectable } from "@nestjs/common";
+import { LocalFileDataStream } from "@src/importer/data-streams/local-file-data-stream";
+
+import { ProductImporter } from "./product-importer";
+
+@Injectable()
+export class ProductImporterService {
+    constructor(private readonly em: EntityManager) {}
+
+    async createImporter(filePath: string) {
+        const importer = new ProductImporter(this.em);
```

Comment:
Why not inject the ProductImporter with DI?

Reply:
I'm not sure if we need any configuration in the constructor soon but I can use DI for now

Reply:
Right now I want a new Instance each time. This should not be injectable then. In the future this might change.